tests/cephfs/snapshot_clone/clone_attributes.py

```diff
@@ -0,0 +1,152 @@
+import logging
+import random
+import string
+import traceback
+
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+
+log = logging.getLogger(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    Test Cases Covered :
+    CEPH-83573524	Ensure the subvolume attributes are retained post clone operations
+
+    Pre-requisites :
+    1. We need atleast one client node to execute this test case
+    2. create fs volume create cephfs if the volume is not there
+    3. ceph fs subvolumegroup create <vol_name> <group_name> --pool_layout <data_pool_name>
+        Ex : ceph fs subvolumegroup create cephfs subvolgroup_clone_attr_vol_1
+    4. ceph fs subvolume create <vol_name> <subvol_name> [--size <size_in_bytes>] [--group_name <subvol_group_name>]
+       [--pool_layout <data_pool_name>] [--uid <uid>] [--gid <gid>] [--mode <octal_mode>]  [--namespace-isolated]
+       Ex: ceph fs subvolume create cephfs subvol_2 --size 5368706371 --group_name subvolgroup_1
+    5. Create Data on the subvolume
+        Ex:  python3 /home/cephuser/smallfile/smallfile_cli.py --operation create --threads 10 --file-size 400 --files
+            100 --files-per-dir 10 --dirs-per-dir 2 --top /mnt/cephfs_fuse1baxgbpaia_1/
+    6. Create snapshot of the subvolume
+        Ex: ceph fs subvolume snapshot create cephfs subvol_2 snap_1 --group_name subvolgroup_clone_attr_vol_1
+    7. Set file and size xattributes on dir
+
+    Test Case Flow:
+    1. Create Clone out of subvolume.
+    2. Mount the cloned volume.
+    3. Validate the contents of cloned volume with contents present of subvolume
+    4. Validate file and size xattributes on dir in cloned volume
+
+    Clean Up:
+    1. Delete Cloned volume
+    2. Delete subvolumegroup
+    """
+    try:
+        fs_util = FsUtils(ceph_cluster)
+        config = kw.get("config")
```

Comment:
Fail if `config` is not provided... `config = kw["config"]`

Reply:
By Default we are getting config value from run.py with some details
config["rhbuild"] = f"{rhbuild}-{platform}" if version2 else rhbuild